### PR TITLE
feat: event: edit list of profiles

### LINKED ISSUE #87: Event: Edit Team
Team:

- List of team members
- Add member -> popup

Add Member:

- Name or Link to profile (similar to venues)
- bio
- role

```
{
    "name": "I'M YOUR DJ New Year 2022",
    "team": [
        {
            "uid": "dfgiouoi234",
            "username": "alejito",
            "name": "Alejito",
            "photo": "https://...",
            "bio": "Dancing since 2007",
            "role": "Instructor",
            "tiktok": "alexrazbakov",
            "instagram": "alexrazbakov",
            "facebook": "alexrazbakov",
            "youtube": "",
        }
    ]
}
```

### PR DESCRIPTION
**Pull request by razbakov**
### User Experience
- edit and view organiser and artists of the event
- change reservation flow

### Developer Experience
- `TInputArray` is a new component that returns array and repeats children as soon as last one is filled.
- `TInputProfile` is an input with autocomplete of profiles from algolia and creation of new FanPage profiles based on Instagram or Facebook usernames.

Demo: https://deploy-preview-89--wedance-razbakov.netlify.app/demo/team/

Closes #87

### CODE REVIEW

File: components/TInput/TInputProfiles.vue [hunk 1]
```diff
@@ -113,7 +113,7 @@ export default {
     const suggestions = computed(() => {
       const { hits } = response.value
       return hits
-        ?.filter((p) => !selected.value.some((sp) => sp.id === p.id))
+        ?.filter((p) => !selected.value.some((sp) => sp.uid === p.id))
```
Comment (best-lucky1030) [inline components/TInput/TInputProfiles.vue:116]: that's `uid` or `uuid` ?

Comment (razbakov) [reply components/TInput/TInputProfiles.vue:116]: uid - user id

Comment (best-lucky1030) [reply components/TInput/TInputProfiles.vue:116]: ah, got it.

**Review by best-lucky1030 — COMMENTED** [review]
checked.

**Pull request merged**